skedge-backend/skegde-test-key-pair

```diff
@@ -0,0 +1,51 @@
+-----BEGIN RSA PRIVATE KEY-----
+MIIJKQIBAAKCAgEApUJMYxpnsztyInKh8UlVf+ZPs50RLCl/LTUi70SwB2PiEjGI
```

Comment:
This is our actual private key, publically available in our github repo?
(I don't mind doing this, I'm just checking that it's what we want to do)

Reply:
Just saw that this is a test value, disregard.